Header header

float32 n1_x
float32 n1_y
float32 n1_z
float32 d1

float32 n2_x
float32 n2_y
float32 n2_z
float32 d2